PolishWithBlondes/src/components/youtube-video/youtube.tsx

```diff
@@ -0,0 +1,59 @@
+import './youtube.scss';
+import { useEffect, useState } from 'react';
+
+interface Video {
+  id: string;
+  snippet: {
+    title: string;
+    thumbnails: {
+      high: {
+        url: string;
+      };
+    };
+  };
+}
+
+export const YoutubeVideos = () => {
+  const [videos, setVideos] = useState<Video[]>([]);
+
+  useEffect(() => {
+    fetch(
+      'https://youtube.googleapis.com/youtube/v3/playlistItems?part=snippet,contentDetails&playlistId=UUlGR6kVvAznpGs_50QlosLQ&key=AIzaSyBaooWVq4y1lsyNk9lLXNt9-ZZG1MjMEXE'
+    )
+      .then((res) => res.json())
+      .then((data) => {
+        setVideos(data.items);
+      })
+      .catch((error) => console.error('Error fetching data:', error));
+  }, []);
+
+  return (
+    <div className="youtube-videos">
+      {videos.map((video, index) =>
+        index < 2 ? (
```

Comment:
If you want to render only two items, maybe it's better to save only 2 items to the state. 
-> Line 25 -> setVideos() - save only two items there and the render everything here without `index < 2` condition